pkg/agent/metrics/prometheus.go

```diff
@@ -67,6 +67,24 @@ var (
 		Help:           "Flow count for each OVS flow table. The TableID is used as a label.",
 		StabilityLevel: metrics.STABLE,
 	}, []string{"table_id"})
+
+	OVSFlowOpsErrorCount = metrics.NewCounterVec(
+		&metrics.CounterOpts{
+			Name:           "antrea_agent_ovs_flow_ops_error_count",
+			Help:           "Number of OVS flow operation errors.",
+			StabilityLevel: metrics.STABLE,
+		},
+		[]string{"operation"},
+	)
+
+	OVSFlowOpsDuration = metrics.NewSummaryVec(
+		&metrics.SummaryOpts{
+			Name:           "antrea_agent_ovs_flow_ops_duration_milliseconds",
+			Help:           "The duration of OVS flow operation",
```

Comment:
```suggestion
			Help:           "The latency of OVS flow operations.",
```

Reply:
Thank you for your comment, I'll fix it.